City-of-Helsinki/drupal-asuntotuotanto
# Asu 292 all images

make fresh (or make cim & make cr)

Added shared apartment images -field to project
Added images -field to apartment
Added computed image field to apartment which combines previously mentioned fields
Added computed field to elastic index as "image_urls"

How to test:
Add one or more images to project's "shared apartment images" and apartment's "images" fields.
Index nodes
Check all those images are indexed.
